src/global/globalEmun.ts

```diff
@@ -0,0 +1,8 @@
+export enum HttpStatus{
+    ERROR=404,
+    SUCCESS=200,
+}
+export enum HttpMessage{
+    ERROR="Server Internal Error",

```

Comment:
nếu status 404 thì message ở đây là Not found mới chuẩn e nè
còn ko thì status Server internal là 500 nhé